plugins/telegram-resources/src/components/Chat.svelte

```diff
@@ -57,18 +57,48 @@
     enabled = res.length > 0
   })
 
-  async function onMessage(event: CustomEvent) {
-    await fetch(url + '/send-msg', {
+  async function onMessage(event: CustomEvent, tryCount = 0) {
+    const to = contactString?.value ?? ''
+    const res = await fetch(url + '/send-msg', {
       method: 'POST',
       headers: {
         Authorization: 'Bearer ' + getMetadata(login.metadata.LoginToken),
         'Content-Type': 'application/json'
       },
       body: JSON.stringify({
-        to: contactString?.value ?? '',
+        to,
         msg: event.detail
       })
     })
+
+    if (res.status === 400 && tryCount === 0) {
```

Comment:
Why we need try count? May be we can just check err.code?

Reply:
Not sure how code may help, it is just a way to exit recursion. Let's call it boolean `isRetry` until we add more complex retrying logic.